### PR TITLE
Lint-staged for pug in tikui

### LINKED ISSUE #10945: Lint-staged for pug in tikui
Following https://github.com/jhipster/jhipster-lite/pull/10935 we should add lint-staged configuration for pug

### PR DESCRIPTION
**Pull request by qmonmert**
Fix #10945

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/generator/client/tikui/domain/TikuiModuleFactory.java [hunk 1]
```diff
@@ -41,6 +41,7 @@ public class TikuiModuleFactory {
   //@formatter:off
   public JHipsterModule buildModule(JHipsterModuleProperties properties) {
     return moduleBuilder(properties)
+      .preCommitActions(stagedFilesFilter("{src/**/,}*.pug"), preCommitCommands("eslint --fix", "prettier --write"))
```
Comment (murdos) [inline src/main/java/tech/jhipster/lite/generator/client/tikui/domain/TikuiModuleFactory.java:44]: Should be sufficient:
```suggestion
      .preCommitActions(stagedFilesFilter("*.pug"), preCommitCommands("eslint --fix", "prettier --write"))
```

Comment (qmonmert) [reply src/main/java/tech/jhipster/lite/generator/client/tikui/domain/TikuiModuleFactory.java:44]: yes it works, I update

File: src/main/java/tech/jhipster/lite/generator/client/tikui/domain/TikuiModuleFactory.java [hunk 2]
```diff
@@ -41,6 +41,7 @@ public class TikuiModuleFactory {
   //@formatter:off
   public JHipsterModule buildModule(JHipsterModuleProperties properties) {
     return moduleBuilder(properties)
+      .preCommitActions(stagedFilesFilter("*.pug"), preCommitCommands("eslint --fix", "prettier --write"))
```
Comment (murdos) [inline src/main/java/tech/jhipster/lite/generator/client/tikui/domain/TikuiModuleFactory.java:44]: I think that the `eslint --fix` is useless until #10943 is done

Comment (qmonmert) [reply src/main/java/tech/jhipster/lite/generator/client/tikui/domain/TikuiModuleFactory.java:44]: ok we will add later

**Review by murdos — APPROVED** [review]

**Pull request merged**